docgen/generate-docs.js

```diff
@@ -116,6 +121,31 @@ function fixLinks(file) {
   });
 }
 
+/** 
+ * Adds links to external documentation for type aliases that
+ * reference an external library.
+ * 
+ * @param data File data to add external library links to.
+ */
+function addTypeAliasLinks(data) {
+  const htmlDom = new JSDOM(data);
+  const fileTags = htmlDom.window.document.querySelectorAll(".tsd-signature-type");
```

Comment:
Not immediately clear why we're selecting `.tsd-signature-type`?

Reply:
I would add a comment here explaining why this tag, so that the next contributor knows the context.

Reply:
Adding what I *think* Erik was doing -- just picking an ID to iterate through that would always be associated with potential external types in TypeDoc.